ast/builtins.go

```diff
@@ -674,6 +686,125 @@ var Sort = &Builtin{
 	),
 }
 
+/**
+ * Type
+ */
+
+// IsNumber takes a string, bool, number value, array or set and
```

Comment:
Since these built-ins can take any kind of value, I would just say `IsNumber returns true if the input value is a number.` (replacing number for the appropriate type below.)